archivers/dzip/Portfile

```diff
@@ -30,8 +30,6 @@ github.tarball_from archive
 
 depends_lib     port:zlib
 
-patchfiles      zlib.patch
```

Comment:
why is this patchfile no longer needed? Does the build now automatically use the MacPorts' installed version of `zlib`?